etc/kayobe/kolla/globals.yml

```diff
@@ -6,13 +6,6 @@
 # non-overcloud hosts
 enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names }}{% endraw %}"
 
-# kolla_base_distro must be set here to be resolvable on a per-host basis
-# This is necessary for os migrations where mixed clouds might be deployed
-kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}"
-
-# Convenience variable for base distro and version string.
-kolla_base_distro_and_version: "{% raw %}{{ kolla_base_distro }}-{{ kolla_base_distro_version }}{% endraw %}"
```

Comment:
I believe these are used for the new auto-generated kolla image tags i.e https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2023.1/etc/kayobe/kolla-image-tags.yml#L7

And in particular:

https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2023.1/etc/kayobe/kolla/globals.yml#L19